tutor/commands/k8s.py

```diff
@@ -46,7 +49,7 @@ def core_api(self):  # type: ignore
         return self._core_api
 
 
-class K8sJobRunner(jobs.BaseJobRunner):
+class K8sTaskRunner(BaseTaskRunner):
```

Comment:
We may want to address the mismatched meaning of "jobs" between Tutor and K8s:
```suggestion
class K8sTaskRunner(BaseTaskRunner):
    """
    Run tasks (bash commands) in Kubernetes-managed services.
    
    Terminology note:
        A Tutor "task" corresponds to a Kubernetes "job".
        In contrast, a "Tutor job" (invoked on the CLI using `tutor k8s do`)
        will trigger one or more Tutor tasks (Kubernetes jobs).
    """
```

Reply:
Good point, I improved the docs.